app/javascript/components/Place.jsx

```diff
@@ -1,21 +1,37 @@
 import React, {useState} from 'react';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
 import {post} from "./useAPI/useAPI";
 import style from '../stylesheets/booking.module.css'
 
 const Place = ({place, token, start_at, end_at, setShowRoom}) => {
-  const [showPlace, setShowPlace] = useState(true)
-  const [error, setError] = useState(null)
+  const [showPlace, setShowPlace] = useState(true);
+  const [error, setError] = useState(null);
+  const [open, setOpen] = useState(false);
+
   const confirmationMessage = `
   Are you sure you want to book 
   a seat number ${place.number} 
-  from ${start_at.replace('T', ' ')} to ${end_at.replace('T', ' ')}
-  `
+  from ${start_at.replace('T', ' ')} to ${end_at.replace('T', ' ')}?
+  `;
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
```

Comment:
```suggestion
  const handleModal = () => setOpen(!open)
```
You can use one function for this feature